binaryop/funcs.go

```diff
@@ -107,3 +107,19 @@ func Ifnot(left, right float64) float64 {
 	}
 	return nan
 }
+
+// And returns 1 if left and right is more than 0. Otherwise 0 is returned.
+func And(left, right float64) float64 {
+	if left > 0 && right > 0 {
+		return left
```

Comment:
How about expression like: `2 and (2 > 1)` ?

Reply:
which return 2. same as origin result.

Reply:
I assume it should return 1 which represent `true`?
```go
// And returns 1 if left and right is more than 0. Otherwise 0 is returned.
```

But we could hold it here and see if valyala and f41gh7 have different opinions.